src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -847,34 +847,31 @@ public void Save(Couchbase.Lite.Linq.IDocumentModel model)
         internal void ResolveConflict([NotNull]string docID, [CanBeNull]IConflictResolver conflictResolver)
         {
             Debug.Assert(docID != null);
+            var saved = false;
```

Comment:
This variable doesn't serve a purpose I feel.  It should be simply ok to move `success` here outside the loop and use it directly.